tensorflow/core/kernels/crop_and_resize_op.cc

```diff
@@ -228,61 +229,54 @@ struct CropAndResize<CPUDevice, T> {
           continue;
         }
 
-        const float height_scale =
-            (crop_height > 1)
-                ? (y2 - y1) * (image_height - 1) / (crop_height - 1)
-                : 0;
-        const float width_scale =
-            (crop_width > 1) ? (x2 - x1) * (image_width - 1) / (crop_width - 1)
-                             : 0;
-
-        for (int y = 0; y < crop_height; ++y) {
-          const float in_y = (crop_height > 1)
-                                 ? y1 * (image_height - 1) + y * height_scale
-                                 : 0.5 * (y1 + y2) * (image_height - 1);
-          if (in_y < 0 || in_y > image_height - 1) {
-            for (int x = 0; x < crop_width; ++x) {
-              for (int d = 0; d < depth; ++d) {
-                crops(b, y, x, d) = extrapolation_value;
-              }
-            }
-            continue;
+        if (method_name == "bilinear") {
+          CachedInterpolation *interp_x = 0l, *interp_y = 0l;
```

Comment:
Our style guides likes one declaration per line, and in the form "CachedInterpolation* interp_x" and we use nullptr instead of 0l for declaring the null pointer.

Also can you make these std::unique_ptrs for memory safety?